package.json

```diff
@@ -9,19 +9,21 @@
     "url": "git@github.com:tcoopman/image-webpack-loader.git"
   },
   "scripts": {
-    "test": "rm -rf test/public/assets && ./node_modules/.bin/webpack --config test/webpack.config.js"
```

Comment:
everything in the `.bin` folder can be accessed directly by npm scripts, no need for the full path `./node_modules/.bin/webpack`. `webpack` is enough.

Also it's best to give descriptive names to different actions.

The test script itself should not care about the cleanup, that's why we have `pre` scripts for :)